p3json/test/test_tool.py

```diff
@@ -54,16 +91,33 @@ def _create_infile(self):
 
     def test_infile_stdout(self):
         infile = self._create_infile()
-        rc, out, err = assert_python_ok('-m', 'json.tool', infile)
-        self.assertEqual(out.splitlines(), self.expect.encode().splitlines())
+        rc, out, err = assert_python_ok('-m', 'pykit.p3json.tool', infile)
+        # self.assertEqual(out.splitlines(), self.expect.encode().splitlines())
+        self._assert_expected(out, linemode=True)
+        self.assertEqual(err, b'')
+
+        # "-" for stdout
+        rc, out, err = assert_python_ok('-m', 'pykit.p3json.tool', infile, '-')
+        # self.assertEqual(out.splitlines(), self.expect.encode().splitlines())
+        self._assert_expected(out, linemode=True)
         self.assertEqual(err, b'')
 
     def test_infile_outfile(self):
         infile = self._create_infile()
         outfile = test_support.TESTFN + '.out'
-        rc, out, err = assert_python_ok('-m', 'json.tool', infile, outfile)
+        rc, out, err = assert_python_ok('-m', 'pykit.p3json.tool', infile, outfile)
         self.addCleanup(os.remove, outfile)
         with open(outfile, "r") as fp:
-            self.assertEqual(fp.read(), self.expect)
+            self._assert_expected(fp.read())
         self.assertEqual(out, b'')
         self.assertEqual(err, b'')
+
+    def test_identical_infile_outfile(self):
+
+        infile = self._create_infile()
+        outfile = infile
+        rc, out, err = assert_python_ok('-m', 'pykit.p3json.tool', infile, outfile)
+        with open(outfile, "r") as fp:
+            self._assert_expected(fp.read())
+
+        self.assertEqual((out, err), (b'', b''))
```

Comment:
Not sure to really understand what you want to test here.
You expect empty?

Reply:
Yes. If it works correctly, `stdout` and `stderr` should be empty. Or error message is printed into one of them.

Reply:
ack